src/main/java/seedu/address/logic/commands/EditMeetingCommand.java

```diff
@@ -0,0 +1,226 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_CLIENT_INDEX;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_DATETIME;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_MEETING_INDEX;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_NAME;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.meeting.Meeting;
+import seedu.address.model.person.Person;
+
+/**
+ * Edits the details of an existing person in the address book.
+ */
+public class EditMeetingCommand extends Command {
+
+    public static final String COMMAND_WORD = "editMeeting";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Edits the details of the meeting identified "
+            + "by the index number used in the displayed meeting list. "
+            + "Existing values will be overwritten by the input values.\n"
+            + "Parameters: INDEX (must be a positive integer) "
+            + PREFIX_CLIENT_INDEX + "CLIENT INDEX "
+            + PREFIX_MEETING_INDEX + "MEETING INDEX "
+            + PREFIX_NAME + "NAME "
+            + PREFIX_DATETIME + "DATETIME \n"
+            + "Example: " + COMMAND_WORD + " "
+            + PREFIX_CLIENT_INDEX + "1 "
+            + PREFIX_MEETING_INDEX + "2 "
+            + PREFIX_NAME + "starbucks meeting "
+            + PREFIX_DATETIME + "01-01-2024 12:00 ";
+
+    public static final String MESSAGE_EDIT_MEETING_SUCCESS = "Edited Meeting: %1$s";
+    public static final String MEETING_NOT_EDITED = "At least one field to edit must be provided.";
+    public static final String MESSAGE_DUPLICATE_MEETING = "This meeting already exists in the address book.";
+
+    private final Index meetingIndex;
+    private final Index clientIndex;
+
+    private final EditMeetingDescriptor editMeetingDescriptor;
+
+    /**
+     * @param clientIndex of the person in the filtered person list to edit
+     * @param meetingIndex of the meeting in the filtered meeting list to edit
+     * @param editPersonDescriptor details to edit the person with
+     */
+    public EditMeetingCommand(Index clientIndex, EditMeetingDescriptor editPersonDescriptor, Index meetingIndex) {
```

Comment:
Maybe your variables could be named differently since it's the meeting we are editing, not the client.